code/datums/martial.dm

```diff
@@ -41,7 +41,7 @@
 		add_logs(A, D, "attempted to [atk_verb]")
 		return 0
 
-	var/obj/item/organ/limb/affecting = D.get_organ(ran_zone(A.zone_sel.selecting))
+	var/obj/item/organ/limb/affecting = D.get_organ(ran_zone(A.zone_selected))
```

Comment:
to replace var/obj/screen/zone_sel, I gave mob a simple var/zone_selected (= "chest")